gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -764,12 +862,17 @@ gst_tensor_filter_common_set_property (GstTensorFilterPrivate * priv,
         }
 
         prop->input_meta.num_tensors = num_dims;
+      } else if (value) {
+        /** Update the properties */
+        if (GST_TF_FW_V0 (priv->fw)) {
+          g_assert (!prop->input_configured);
+        } else {
+          /** TODO: update the properties */
```

Comment:
You will need to re-negotiate the GST-Caps if you allow this.

Reply:
Yes, this is removed for now. Re-negotiation can be supported later.